src/DmaFactory.cpp

```diff
@@ -114,11 +114,6 @@ extern "C" void DMAC_4_Handler(void)
 }
 #endif
 
-/**
- * Base implementation of a DMA callback
- */
-void DmaComponent::dmaTransferComplete(DmaCode) {}
-
 DmaControllerInstance::DmaControllerInstance()
```

Comment:
this just leads to trouble and everyone implements it anyways

Reply:
Yeah, it should have been left at an abstract interface.